snapcraft/plugins/ant.py

```diff
@@ -59,22 +102,88 @@ def schema(cls):
             "items": {"type": "string"},
             "default": [],
         }
+
+        schema["properties"]["ant-version"] = {"type": "string"}
+
+        schema["properties"]["ant-version-checksum"] = {"type": "string"}
+
+        schema["required"] = ["source"]
+        schema["properties"]["ant-openjdk-version"] = {"type": "string", "default": ""}
```

Comment:
This default isn't actually necessary, is it?

Reply:
just copying from maven